{
    "type": "bundle",
    "id": "bundle--86763ba8-f391-49ca-9e78-b5ef44cc03ab",
    "spec_version": "2.0",
    "objects": [
        {
            "object_marking_refs": [
                "marking-definition--fa42a846-8d90-4e51-bc29-71d5b4802168"
            ],
            "type": "relationship",
            "id": "relationship--96417d00-2a6b-4a22-b43c-f65cb60da952",
            "description": "[Pysa](https://attack.mitre.org/software/S0583) has used [PsExec](https://attack.mitre.org/software/S0029) to copy and execute the ransomware.(Citation: CERT-FR PYSA April 2020)",
            "target_ref": "attack-pattern--f1951e8a-500e-4a26-8803-76d95c4554b4",
            "modified": "2021-04-27T03:33:35.425Z",
            "source_ref": "malware--a19c1197-9414-46e3-986f-0f609ff4a46b",
            "created_by_ref": "identity--c78cb6e5-0c4b-4611-8297-d1b8b55e40b5",
            "relationship_type": "uses",
            "created": "2021-03-02T16:42:09.554Z",
            "external_references": [
                {
                    "url": "https://www.cert.ssi.gouv.fr/uploads/CERTFR-2020-CTI-003.pdf",
                    "description": "CERT-FR. (2020, April 1). ATTACKS INVOLVING THE MESPINOZA/PYSA RANSOMWARE. Retrieved March 1, 2021.",
                    "source_name": "CERT-FR PYSA April 2020"
                }
            ]
        }
    ]
}